index.html

```diff
@@ -20,18 +20,18 @@
 
     <div class="col-sm-10 col-sm-offset-1">
       <div class="row">
-        <div class="col-sm-10">
+        <div class="col-sm-9">
           <h1 class="pull-left">space barnacle <small>thinking inside the box</small></h1>
```

Comment:
Is it possible to set it so that it changes colors when the user hovers over it?